### PR TITLE
feat:add private and drafts share paths

### PR DESCRIPTION
**Pull request by piyushagwl007**
## PR Checklist
Please check that your PR fulfills the following requirements:

- [x] The commit message follows [our guidelines](https://github.com/pesto-students/batch-9-Lumen/wiki/Commit-message-convention)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] GIFs/Videos/Images added (only for frontend changes)

## PR Type
What kind of change does this PR introduce? 

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Infrastructure changes
- [ ] Other... Please describe:
Issue Number: 
#

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: server/src/services/blogs/render.js [hunk 1]
```diff
@@ -1,26 +1,30 @@
...
-  
+  const {
+    _id,
+    title = '',
+    description = '',
+    imageUrl = 'https://cdn3.wpbeginner.com/wp-content/uploads/2016/11/blogimagetools.jpg'
+  } = req.blog;
+  const prependPath = `${env.appViewUrl}blog`;
+  const draftPath = `${prependPath}/draft/${req.params.draftUrl}/${_id}`;
+  const privatePath = `${prependPath}/secured/${req.params.draftUrl}/${_id}`;
```
Comment (nihalanisumit) [inline server/src/services/blogs/render.js]: Fix this error.

File: server/src/services/blogs/render.js [hunk 2]
```diff
@@ -1,26 +1,30 @@
...
+  const publicUrl = `${prependPath}/${_id}`;
+  let redirectUrl;
+  if (req.params.draftUrl) {
+    redirectUrl = draftPath;
+  } else if (req.params.draftUrl) {
```
Comment (nihalanisumit) [inline server/src/services/blogs/render.js]: fix this too

**Review by nihalanisumit — APPROVED** [review]

File: server/src/services/blogs/render.js (hunk 1 above)
Comment (piyushagwl007) [reply server/src/services/blogs/render.js]: Can you elaborate what error are you talking about ? :smile:

File: server/src/services/blogs/render.js (hunk 2 above)
Comment (piyushagwl007) [reply server/src/services/blogs/render.js]: Can you elaborate what error are you talking about ? :smile:

**Pull request merged**